frontend/src/pages/Graphing/Dashboard.tsx

```diff
@@ -35,6 +44,10 @@ export const Dashboard = () => {
 
 	const navigate = useNavigate()
 
+	const [deleteGraph, deleteContext] = useDeleteGraphMutation({
```

Comment:
Looks like `deleteContext` is not used - may be causing a build error? Same with `idx` below